lib/cmds/init.ts

```diff
@@ -76,6 +77,15 @@ export const init = async () => {
       })
     }
   }
+
+  // something something, get the flow type
+  // one of 'Contentful.js', 'GraphQL', 'REST API'
+  const connectionType = 'GraphQL'
```

Comment:
Yup, we need another prompt here to choose one of ....

Reply:
Wouldn't that come from previously in the flow?